docs/source/input_preparation.rst

```diff
@@ -0,0 +1,115 @@
+
+Protonating inputs
+==================
+
+All files in the Schrodinger input files are considered to be in their intended protonation state. No additional protonation steps should be carried out.
+
+.. note::
+   1. Care should be taken that any processing steps do not alter the protonation state. For example, PyMol is known to change histidines from HID to HIE during capping.
+
+   2. To ensure that protonation states were not altered, please ensure that the prepared PDB file has the same number of protein atoms outside of capping groups.
+
+Setting residue names
+=====================
+
+Where possible, residue should be assigned PDB-compliant names.
+
+*Example 1: Waters named SPC (e.g. in the case of Thrombin in the JACS set), should be renamed to HOH.*
+
+*Example 2: Capping groups named NMA should be renamed to NME (e.g. in the case of PTP1B in the JACS set).*
+
+Fixing hydrogen atom names
+==========================
+
+In some cases, hydrogen names may need to be manually altered to match expected, i.e. PDB compliant names.
+
+These exact cases can be difficult to identify, running the validation script (see below), will help identify these. Please reach out to the OpenFE team should you encounter any unknown hydrogen names.
+
+*Example 1: GLY termini hydrogens being named 3HA and HA instead of HA3 and HA2.*
+
+*Example 2: HIS (in the HID state) hydrogens being named 1HD, 2HD, and 1HE instead of HD1, HD2, and HE1.*
+
+Validating prepared files
+=========================
+
+To ensure that prepared files can be run using OpenFE, a short MD simulation validation script has been provided: <insert location of the script>. In an environment with OpenFE 1.0 installed, please run this script by calling:
+
+.. code-block:: python
+
+   # If you don’t have cofactors
+   python validate-inputs --pdb protein.pdb
+
+   # If you have cofactors
+   python validate-inputs --pdb protein.pdb --cofactors cofactors.sdf
+
+
+If the script outputs “SIMULATION COMPLETE”, then your inputs are suitable for use with OpenFE. If they do not, then there is likely an issue with the input file. Please report the error message emitted when contacting the OpenFE team for advice on how to fix any issues.
+
+.. note::
+   This script runs a very short simulation, it is recommended that it is executed on a machine with a CUDA-enabled GPU.
+
+Submitting prepared input files
+===============================
+
+All prepared inputs should be submitted to the OpenFE Public Benchmark github repository. This should be done via Pull Request, with a folder for each prepared system including the protein PDB, ligand SDF, relevant edges CSV, and if available cofactor SDF file. A short bullet point summary of any remediation steps, including any software used, should also be included as a markdown file.
```

Comment:
Probably worth here linking to a hypothetical proposed "contributing inputs" page.